### PR TITLE
7390 block v3 flag

### PR DESCRIPTION
**Pull request by mehdi-aouadi**
## PR Description
Add `--Xblock-v3-enabled` to use block v3 for block production

## Fixed Issue(s)

#7529 

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java [hunk 1]
```diff
@@ -83,6 +84,7 @@ public class ValidatorConfig {
   private final boolean doppelgangerDetectionEnabled;
   private final boolean failoversSendSubnetSubscriptionsEnabled;
   private final boolean failoversPublishSignedDutiesEnabled;
+  private final boolean blockV3enabled;
```
Comment (zilm13) [inline validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java]: `blockV3Enabled` here and in next places would be more correct on case by my feelings

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java [hunk 2]
```diff
@@ -410,7 +410,8 @@ private void scheduleValidatorsDuties(
             forkProvider,
             validatorApiChannel,
             blockContainerSigner,
-            config.getValidatorConfig().isBlindedBeaconBlocksEnabled(),
+            config.getValidatorConfig().isFailoversPublishSignedDutiesEnabled(),
```
Comment (zilm13) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: shouldn't it be `isBlindedBeaconBlocksEnabled()`?

Comment (mehdi-aouadi) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: My bad! Fixed it

File: validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java (hunk 1 above)
Comment (mehdi-aouadi) [reply validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java]: Definitely! Fixed

**Review by zilm13 — APPROVED** [review]
LGTM

**Pull request merged**